agent.go

```diff
@@ -733,11 +733,22 @@ func (s *sandbox) listenToUdevEvents() {
 
 			// Notify watchers that are interested in the udev event.
 			// Close the channel after watcher has been notified.
-			for devPCIAddress, ch := range s.deviceWatchers {
-				if ch != nil && strings.HasPrefix(uEv.DevPath, filepath.Join(rootBusPath, devPCIAddress)) {
+			for devAddress, ch := range s.deviceWatchers {
+				if ch == nil {
+					continue
+				}
+
+				if
```

Comment:
Errr. This looks really odd to me - have you passed this through `gofmt`? :smile:

Reply:
Hi @jodh-intel , I know it's  look odd, but it's really passed  the gofmt. :)  
Now I changed the style by "goto",  maybe it looks much better.